NEWS.md

```diff
@@ -1,6 +1,7 @@
 ## 22.1.0 IN-PROGRESS
 
 * Added Administrative notes to instance records (MODINVSTOR-832)
+* Added administrative notes to holdings records (MODINVSTOR-833)
```

Comment:
I think these lines might as well be combined into one

Reply:
I'm about to do a PR for the item records-I can consolidate them in that one.